pkg/harvester-manager/list/harvesterhci.io.management.cluster.vue

```diff
@@ -62,6 +62,7 @@ export default {
 
     this.hciClusters = hash.hciClusters;
     this.mgmtClusters = hash.mgmtClusters;
+    this.kubeVersion = hash.mgmtClusters.find((c) => c.id === 'local')?.kubernetesVersionBase || '';
```

Comment:
this would be a minor performance issue.

there's a getter to find a specific resource by id (the cache has all resources mapped by id). this it's <store>/byId